### PR TITLE
File Sharing UI - Making components consistent with Figma

### PR DESCRIPTION
**Pull request by anjulgarg**
# What
Modifies file sharing  components to make them consistent with FIGMA 
https://www.figma.com/file/72qJ03EOQExf12g5JTdWjQ/Web-UI-Library?node-id=4022%3A24730

# Why

# How Tested
Manually

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/SendBox.tsx [hunk 1]
```diff
@@ -305,18 +305,18 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
...
-    [props]
-  );
+  const onRenderFileUploads = useCallback(() => {
+    if (props.activeFileUploads?.length === 0) {
+      return null;
+    }
+    return props.onRenderFileUploads ? (
+      props.onRenderFileUploads()
+    ) : (
+      <div style={{ margin: '0.25rem' }}>
```
Comment (JamesBurnside) [inline packages/react-components/src/components/SendBox.tsx:315]: From the snapshot of the sendbox, this looks to always be being applied even when no fileUploads are present?
image

This looks to be causing the sendbox being slightly too big when used just as a component (outside of a composite):
image

Comment (anjulgarg) [reply packages/react-components/src/components/SendBox.tsx:315]: Yes, pushed another commit to fix this. :)

Comment (anjulgarg) [reply packages/react-components/src/components/SendBox.tsx:315]: Fixed

File: packages/react-components/src/components/SendBox.tsx [hunk 2]
```diff
@@ -305,18 +305,18 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
...
-    [props]
-  );
+  const onRenderFileUploads = useCallback(() => {
+    if (!props.activeFileUploads?.length) {
+      return null;
+    }
+    return props.onRenderFileUploads ? (
+      props.onRenderFileUploads()
+    ) : (
+      <div style={{ margin: '0.25rem' }}>
```
Comment (JamesBurnside) [inline packages/react-components/src/components/SendBox.tsx:315]: Looks like we should avoid setting this margin when there are active file uploads, but they failed (or there is some other logic needed to remove them from props.activeFileUploads):
image

Comment (anjulgarg) [reply packages/react-components/src/components/SendBox.tsx:315]: Fixed, thanks for pointing out.

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**